openapitools.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "./node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "spaces": 2,
+  "generator-cli": {
+    "version": "7.14.0"
```

Comment:
Checking in this file, which gets auto-generated when the client is built if it doesn't already exist. This lets us pin to a specific version of open api generator. I ran into a lot of problems stemming from this not being pinned and (reasons) preventing my environment from asking Maven for the latest version.